packages/ra-auth-cognito-language-french/package.json

```diff
@@ -0,0 +1,59 @@
+{
+    "name": "ra-auth-cognito-language-french",
+    "version": "1.1.0",
+    "repository": "git@github.com:marmelab/ra-auth-cognito.git",
+    "author": "Guillaume Pierson <guillaumep@marmelab.com> (https://marmelab.com/)",
+    "license": "MIT",
+    "files": [
+        "*.md",
+        "lib",
+        "esm",
+        "src"
+    ],
+    "main": "lib/index",
+    "module": "esm/index.js",
+    "types": "esm/index.d.ts",
+    "sideEffects": false,
+    "peerDependencies": {
+        "ra-auth-cognito": "^1.1.0",
+        "react-admin": "^4.16.9"
```

Comment:
I agree on the conclusion (we do not need this peerDep) but not on the rationale: we do not need react-admin in this package because none of its code uses react-admin.